content/running-amazon-ec2-workloads-at-scale/deploy_lb.md

```diff
@@ -11,64 +11,66 @@ Each target group routes requests to one or more registered targets, such as EC2
 
 1. Execute the following command to edit **application-load-balancer.json** and update the values with the resources created by Cloudformation.
 
-	```
-	sed -i.bak -e "s/%publicSubnet1%/$publicSubnet1/g" -e "s/%publicSubnet2%/$publicSubnet2/g" -e "s/%loadBalancerSecurityGroup%/$loadBalancerSecurityGroup/g" application-load-balancer.json
-	```
+    ```bash
+    sed -i.bak -e "s/%publicSubnet1%/$publicSubnet1/g" -e "s/%publicSubnet2%/$publicSubnet2/g" -e "s/%loadBalancerSecurityGroup%/$loadBalancerSecurityGroup/g"    application-load-balancer.json
```

Comment:
remove the extra  space between loadBalancerSecurityGroup/g"    application-load-balancer.json.

Reply:
Removed in 47aeaac